src/Individual.php

```diff
@@ -40,7 +41,7 @@ public function getSurname()
 
     public function setSurname($value)
     {
-        $this->surname = $value;
+        $this->surname = substr($value, 0, 35);
```

Comment:
I think it's probably worth using mb_substr and declaring the dependency on `ext-mbstring` in composer.json. No need to have a new major version number if we don that (https://github.com/Ocramius/ProxyManager/issues/464#issuecomment-504590589 ).